### ISSUE TITLE
Chromedriver error

### ISSUE THREAD

**Issue by sseerrggii** [coopdevs/timeoverflow-provisioning#67]
Reported by Jordi in community you can't Build development environment on

`ansible-playbook playbooks/provision.yml --limit=dev 
`

return error:

```
TASK [common : Symlink chromedriver globally] **********************************
fatal: [local.timeoverflow.org]: FAILED! => {"changed": false, "gid": 0, "group": "root", "mode": "0755", "msg": "refusing to convert from file to symlink for /usr/bin/chromedriver", "owner": "root", "path": "/usr/bin/chromedriver", "size": 12375944, "state": "file", "uid": 0}
```

**Comment by enricostano** [comment]
@markets @Morantron could be related to https://github.com/coopdevs/timeoverflow/pull/491 ?

**Comment by Morantron** [comment]
Looks like they fixed the ubuntu package, and the symlink is not needed anymore

http://changelogs.ubuntu.com/changelogs/pool/universe/c/chromium-browser/chromium-browser_73.0.3683.86-0ubuntu0.16.04.1/changelog

```
chromium-browser (72.0.3626.109-0ubuntu0.16.04.1) xenial; urgency=medium

  * Upstream release: 72.0.3626.109
  * debian/rules:
    - restore old keepalive snippet to prevent builds from timing out during
      the link phase (this happens often enough on armhf, Launchpad builders
      have an inactivity timeout of 150 minutes)
>>> - install the chromedriver executable in /usr/bin, where python{,3}-selenium
      and other packages expect it by default (LP: #1667208) <<<
  * debian/control: make chromium-chromedriver provide "chromium-driver"
  * debian/chromium-chromedriver.{dirs,install}: removed, no longer needed
  * debian/tests/{chromium-version,html5test}: remove custom driver path
```

This PR could be reopen and merged https://github.com/coopdevs/timeoverflow-provisioning/pull/66/files

**Comment by enricostano** [comment]
@sseerrggii you were right! :joy: :point_up_2:

**Comment by enricostano** [comment]
> This PR could be reopen and merged

Well... I would try first that feature specs still work :sweat_smile:

**Comment by enricostano** [comment]
Fixed by #66
